front/app/containers/Admin/pagesAndMenu/containers/CustomPages/New/NewCustomPage.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import CustomPageSettingsForm from '../CustomPageSettingsForm';
+import { FormValues } from 'containers/Admin/pagesAndMenu/containers/CustomPages/CustomPageSettingsForm';
+import { createCustomPage } from 'services/customPages';
+import clHistory from 'utils/cl-router/history';
+
+const NewCustomPage = () => {
+  const handleOnSubmit = async (formValues: FormValues) => {
+    const { data } = await createCustomPage(formValues);
+
+    setTimeout(
+      () => clHistory.push(`/admin/pages-menu/custom/${data.id}/content`),
+      1500
+    );
+  };
```

Comment:
Added this because the redirect was too sudden (with a short flash of the success message.

Reply:
👍 agree with adding it, much better UX, would just suggest a comment in the code as well since I can imagine a future dev thinking it's a bug workaround of some kind instead of a purposeful UX decision